dashboard/src/components/entitypage/EntityPageView.js

```diff
@@ -516,11 +516,21 @@ const EntityPageView = ({
                       )}
                     {metadata['error'] && metadata['error'] !== '' && (
                       <>
-                        <b>Error Message:</b>
-                        <ErrorModal
-                          buttonTxt='See More'
-                          errorTxt={metadata['error']}
-                        />
+                        <strong>Error Message:</strong>
```

Comment:
This doesnt seem to be picking up newlines in the error message

Reply:
I'll check it out.